lighttpd.conf

```diff
@@ -0,0 +1,46 @@
+mimetype.assign = (
+  ".html" => "text/html",
+  ".htm" => "text/html",
+  ".shtml" => "text/html",
+  ".css" => "text/css",
+  ".gif" => "image/gif",
+  ".jpeg" => "image/jpeg",
+  ".jpg" => "image/jpeg",
+  ".js" => "application/javascript",
+  ".txt" => "text/plain",
+  ".png" => "image/png",
+  ".ico" => "image/x-icon",
+  ".svg" => "image/svg+xml",
+  ".webp" => "image/webp",
+  ".json" => "application/json",
+)
+
+server.username = "www-data"
+server.groupname = "www-data"
+
+server.max-fds = 8192
+server.max-connections = 4096
```

Comment:
ditto. defaults to 1024; is there some indication suggesting we should use a greater number? I am not saying I know we should not. I ask, because I would very much like to choose good values based on some data-source. in doubt, I would probably stick w/ default values.

in case this file was generated (from build / lighttps), it might be a good idea to state so in a comment, so it is clear those are just default-values from lighttpd, rather than ours.